cloudbuild/unit_test_jobs.jsonnet

```diff
@@ -0,0 +1,36 @@
+local base = import 'templates/base.libsonnet';
+local gpus = import 'templates/gpus.libsonnet';
+
+local image = std.extVar('image');
+local tagName = std.extVar('tag_name');
+local gcsBucket = std.extVar('gcs_bucket');
+
+local unittest = base.BaseTest {
```

Comment:
Does unittest have to be camel case here? Also entrypoint later on

Reply:
Honestly i don't know! I am just following the tutorial here: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/tutorials/doc/tutorials/basic/mnist-gpu.jsonnet